src/main/kotlin/ai/digital/integration/server/common/domain/Database.kt

```diff
@@ -6,7 +6,7 @@ import org.gradle.kotlin.dsl.property
 @Suppress("UnstableApiUsage")
 open class Database(objects: ObjectFactory) {
 
-    var derbyPort: Int = objects.property<Int>().value(1527).get()
+    var databasePort: Int = objects.property<Int>().value(1527).get()
```

Comment:
It's not really correct now, as it depends on the database. 1527 is only for derby